esphome/components/inkbird_ibsth1_mini/inkbird_ibsth1_mini.cpp

```diff
@@ -73,6 +73,10 @@ bool InkbirdIbstH1Mini::parse_device(const esp32_ble_tracker::ESPBTDevice &devic
 
   // Read bluetooth data into variable
   auto measured_temperature = mnf_data.uuid.get_uuid().uuid.uuid16 / 100.0f;
+  // The temperature rolls back around to 65535 / 100.0f = 655.35 when it reaches zero so adjust accordingly
+  if (measured_temperature > 327.68) {
+    measured_temperature = measured_temperature - 655.35;
+  }
```

Comment:
```suggestion
  auto measured_temperature = ((int16_t) mnf_data.uuid.get_uuid().uuid.uuid16) / 100.0f;
```

That looks like it's just 16bit 2's complement, so this should work too (and doesn't depend on floating point math)

Reply:
yes, it's a type change, see https://github.com/esphome/esphome/pull/2470, seems like we did duplicate fixes. :-)